Xamarin.Forms.Core/ExpanderOrientationExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Xamarin.Forms
+{
+	public static class ExpanderOrientationExtensions
```

Comment:
I could make it internal or even remove, but I think it would be useful for users

Reply:
Made internal for now